tests/inputs/TestConfig.cpp

```diff
@@ -22,6 +22,7 @@ TEST(Config, Nominal) {
   ASSERT_TRUE(config.useVSCAsGenerators());
   ASSERT_TRUE(config.useLCCAsLoads());
   ASSERT_EQ("/tmp", config.outputDir());
+  ASSERT_EQ(45.0, config.getDsoVoltageLevel());
```

Comment:
The purpose of the nominal test is to check that a value different from the default is taken into account